keyboards/beekeeb/3w6hs/info.json

```diff
@@ -0,0 +1,61 @@
+{
+    "manufacturer": "beekeeb",
+    "keyboard_name": "3w6hs",
+    "maintainer": "beekeeb",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0003",
+        "vid": "0xBEEB"
+    },
```

Comment:
```suggestion
        "vid": "0xBEEB",
        "no_startup_check": true
    },
    "matrix_pins": {
        "custom_lite": true
    },
```

Reply:
@dunk2k  When `"custom_lite": true` is used instead of `CUSTOM_MATRIX = lite`, I got the errors ```
☒ beekeeb/3w6hs: MATRIX_COLS is inconsistent with the size of MATRIX_COL_PINS: 0 != 10
☒ beekeeb/3w6hs: MATRIX_ROWS is inconsistent with the size of MATRIX_ROW_PINS: 0 != 8
```